app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -573,3 +623,69 @@ public void onError(String errorMessage) {
         }
     }
 }
+
+
+class PdfAndroidJavascriptBridge {
```

Comment:
I don't normally put 2 classes at the root of same file, maybe make this a static inner class or move it to a separate file called `PdfAndroidJavascriptBridge.java`?

Reply:
I'll move to a static inner class